packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -131,8 +128,6 @@ class GuildBanManager extends CachedManager {
   /**
    * Options used to ban a user from a guild.
    * @typedef {Object} BanOptions
```

Comment:
With the removal of `deleteMessageDays`, `BulkBanOptions` is equivalent to `BanOptions`. Perhaps we should remove one of them